cmd/kube-scheduler/app/server.go

```diff
@@ -329,10 +329,3 @@ func newHealthzHandler(config *kubeschedulerconfig.KubeSchedulerConfiguration, s
 	}
 	return pathRecorderMux
 }
-
-// WithPlugin creates an Option based on plugin name and factory.
-func WithPlugin(name string, factory framework.PluginFactory) Option {
-	return func(registry framework.Registry) error {
-		return registry.Register(name, factory)
-	}
-}
```

Comment:
this will be used by out-of-tree plugins.

Reply:
please add a comment to say that "This function is used register out-of-tree plugins"

Reply:
Thanks your advice.

Reply:
done